web/views/note_view.ex

```diff
@@ -1,4 +1,12 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  defp sanitize_html(unsafe) do
+    {:safe, HtmlSanitizeEx.html5(unsafe)}
```

Comment:
The documentation is a little shoddy, so I looked at the tests. `basic_html` strips nested tags, and I didn't think that was necessary. Didn't look any further, but it seemed like `html5` simply sanitizes and nothing more which is what I want.